### DISCUSSION TITLE
Working with day button identifiers following recent accessibility changes

### DISCUSSION THREAD

**Discussion by LouisYoungGMSL** [gpbl/react-day-picker#1716] [General]
Hi 👋 

Firstly, thanks for the awesome library. 

I wanted to open a discussion surrounding the latest accessibility changes. Previously, each day cell button used to have an accessible name that was the formatted day and month long name, making it easy to identify days for users using assistive technologies, and for automation testing. In the latest version, this has been replaced with just the day number (https://github.com/gpbl/react-day-picker/commit/128ec550df06b4098e86ec575bb88f3115ca5691#diff-563d7b9b7c6dbe2873cce6b656876b7efb60664516ff5b1978e929cb467c11c6L6). 

I've found this new behaviour to be problematic during automated testing when looking at a calendar showing an overlap of two months where the same day number occurs twice, as I can no longer identify which month the day belongs to. With the new implementation, they now have identical names. I also wonder whether this will be difficult for users using assistive technologies.

Do you have a suggestion for how to identify the day unambiguously? I can't see a way to do this with the current implementation. 

Thanks 🙂

**Comment by gpbl** [comment]
Hi @LouisYoungGMSL, yes that looks like an oversight from my side. I have been testing DayPicker with JAWS screen reader and "it looked fine". Seeing the amount of feedback, I was wrong. We will revert the change and fix the grid for accessibility. 

@Kosai106 already started the work in https://github.com/gpbl/react-day-picker/pull/1708.  Meanwhile, you could fix to `v8.4`. Thanks for the understanding and the feedback!

**Reply by LouisYoungGMSL** [reply]
Thank you, that sounds ideal 🙌
